2021/omips-online/OMIPS-2021-Barrankas/settings.json

```
@@ -0,0 +1,28 @@
+{
+  "title": "Barrankas",
+  "source": "OMIPS 2021 (César Cepeda)",
+
+  "limits": {
+    "TimeLimit": 1000,
+    "MemoryLimit": 33554432,
+    "InputLimit": 33554432,
+    "OutputLimit": 16384,
+    "ExtraWallTime": 0,
+    "OverallWallTimeLimit": 60000
+  },
+
+  "validator": {
+    "name": "token-caseless",
+    "limits": {
+      "TimeLimit": 1000
+    }
+  },
+
+  "misc": {
+    "alias": "OMIPS-2021-Barrankas",
+    "visibility": "private",
```

```suggestion
    "visibility": "public",
```
Hay que cambiar estos, no? Supongo lo podemos hacer en otro PR.